.rubocop.yml

```diff
@@ -248,6 +248,8 @@ Style/KeywordParametersOrder:
   Enabled: false
 Style/MethodDefParentheses:
   Enabled: false
+Style/NumericPredicate:
+  Enabled: false
```

Comment:
@eanders is this okay with you? It means both `x > 0` and `x.positive?` will pass rubocop